CMake/Version.cmake

```diff
@@ -1,11 +1,28 @@
+set(VERSION_MAJOR 0)
```

Comment:
lets set these to more reasonable defaults, like `1.12`. Current `git describe` gives me `v1.12.0-22-g40ba170c`, so maybe use that? That way if anything uses these variables it will look sensible.

We can also update this version as part of https://github.com/google/flatbuffers/wiki/FlatBuffers-release-process

Reply:
If we are fine with a manual update of these variables each time a new version is released, then I will do this, no problem.

Reply:
Done

Reply:
Unfortunately, I see no edit button for the aforementioned wiki page. Will you add this file to the list of files to update when releasing a new version?